server.js

```diff
@@ -68,7 +68,15 @@ client.on('message', (message) => {
   }
 
   if (message.content.replace(/\s/g, '').match(IS_NESTCEPAS_REGEX)) {
-    message.channel.send('', { file: `${PATH_TO_STATIC_FOLDER}/nestcepas.gif` });
+    message.channel.send('', {
```

Comment:
Not very fond of the code style here, semi inline and line return. Ssomething more coherent would be appreciated.

Reply:
Dunno why this line was split